launcher/minecraft/WorldList.cpp

```diff
@@ -222,7 +234,16 @@ QVariant WorldList::data(const QModelIndex &index, int role) const
         }
 
     case Qt::ToolTipRole:
-    {
+    {   
+        if (column == InfoColumn) {
+            if (world.isSymLinkUnder(instDirPath())) {
+                return tr("Warning: This world is symbolicly linked from elsewhere. Editing it will also change the origonal") +
```

Comment:
You mean symbolically linked and editing will also change the original?

Reply:
```suggestion
                return tr("Warning: This world is symbolically linked from elsewhere. Editing it will also change the original file.") +
```